cmd/target-browser/cmd/get.go

```diff
@@ -82,5 +85,12 @@ func validateInput(cmd *cobra.Command) error {
 		return fmt.Errorf("[%s] flag invalid value. Usage - %s", OutputFormatFlag, OutputFormatFlagUsage)
 	}
 
+	if operationScope != "" {
+		val, isPresent := internal.ResourceScopeMap[strings.ToLower(operationScope)]
```

Comment:
By default, filter `operationScope` returns both  `singleNamespace` and `MultiNamespace` resources
so that I have not added default value `singleNamespace` 
can I add default value of flag `singleNamespce` ?